nntrainer/nntrainer_logger.cpp

```diff
@@ -65,7 +65,13 @@ Logger::Cleanup::~Cleanup() {
   Logger::ainstance = nullptr;
 }
 
-Logger::~Logger() { outputstream.close(); }
+Logger::~Logger() {
+  try {
+    outputstream.close();
+  } catch(...) {
```

Comment:
This is to pass C++ format checker 
```suggestion
  } catch (...) {
```